src/Listener/SubSelectPalettesListener.php

```diff
@@ -156,6 +164,11 @@ private function getValue($dataContainer, $strTable, $strSelector)
     {
         $strValue = null;
 
+        if (method_exists($dataContainer, 'getModel')) {
```

Comment:
We should use an explicit check `$dataContainer instanceof ContaoCommunityAlliance\DcGeneral\Contao\Compatibility\DcCompat` here

Reply:
do I have to integrate this via the controller?
https://github.com/contao-community-alliance/meta-palettes/pull/44/files#diff-fd599f43b9cf27fe5cfe7615160733e923d72ad3067f8c8163b59c32740261d8R168